From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: zador-blood-stained <zador-blood-stained@users.noreply.github.com>
Date: Thu, 17 Aug 2017 20:00:34 +0300
Subject: [ARCHEOLOGY] Set MAC addresses for Rock64 from armbianEnv.txt

> X-Git-Archeology: > recovered message: > Currently rockchip-cpuinfo driver reads SoC serial as all zeroes due to
> X-Git-Archeology: > recovered message: > either a bug, hardware limitation or serial not flashed to eFuses
> X-Git-Archeology: - Revision d5c6bf69db29fd8cd0b929aa15d7fc056cb02441: https://github.com/armbian/build/commit/d5c6bf69db29fd8cd0b929aa15d7fc056cb02441
> X-Git-Archeology:   Date: Thu, 17 Aug 2017 20:00:34 +0300
> X-Git-Archeology:   From: zador-blood-stained <zador-blood-stained@users.noreply.github.com>
> X-Git-Archeology:   Subject: Set MAC addresses for Rock64 from armbianEnv.txt
> X-Git-Archeology:
> X-Git-Archeology: - Revision 62e07d2ccad712e4c3448fbc56a0d417637d7646: https://github.com/armbian/build/commit/62e07d2ccad712e4c3448fbc56a0d417637d7646
> X-Git-Archeology:   Date: Sun, 27 Aug 2017 22:52:05 +0200
> X-Git-Archeology:   From: Igor Pecovnik <igor.pecovnik@gmail.com>
> X-Git-Archeology:   Subject: RK3328 remove deprecated patches, config update
> X-Git-Archeology:
> X-Git-Archeology: - Revision c63b2129f64a741d71b30bc7bf3bb6ede98de6fb: https://github.com/armbian/build/commit/c63b2129f64a741d71b30bc7bf3bb6ede98de6fb
> X-Git-Archeology:   Date: Sun, 01 Jul 2018 13:55:11 +0000
> X-Git-Archeology:   From: Igor Pecovnik <igor.pecovnik@gmail.com>
> X-Git-Archeology:   Subject: Lock RK3328 to last known working tag. Thanks for idea/tip to @markh-de
> X-Git-Archeology:
> X-Git-Archeology: - Revision 41b589372f2c1a8450033db78f3d8ed9db1b2196: https://github.com/armbian/build/commit/41b589372f2c1a8450033db78f3d8ed9db1b2196
> X-Git-Archeology:   Date: Sun, 26 Aug 2018 14:14:14 +0200
> X-Git-Archeology:   From: Igor Pecovnik <igor.pecovnik@gmail.com>
> X-Git-Archeology:   Subject: Creating new family rockchip64. 3288 and 3399 was merged in, except Nanopi which stays in RK3399 family with sources attached to FA repository until we manage to join that in. Enabled Nanopc T4, Nanopi M4, Renegade. Tested on Rock64 and Nanopc T4.
> X-Git-Archeology:
> X-Git-Archeology: - Revision 150ac0c2afa147d9e3b036c8ecd8238fe5648cf3: https://github.com/armbian/build/commit/150ac0c2afa147d9e3b036c8ecd8238fe5648cf3
> X-Git-Archeology:   Date: Tue, 19 Nov 2019 23:25:39 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Remove K<4, change branches, new features (#1586)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 0cdffb29b07305209efb12cf3b5ac6032d3a1153: https://github.com/armbian/build/commit/0cdffb29b07305209efb12cf3b5ac6032d3a1153
> X-Git-Archeology:   Date: Wed, 24 Mar 2021 19:01:53 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Renaming DEV branch to EDGE (#2704)
> X-Git-Archeology:
---
 arch/arm64/boot/dts/rockchip/rk3328-rock64.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3328-rock64.dts b/arch/arm64/boot/dts/rockchip/rk3328-rock64.dts
index 8f39efce33cc..48a6c74cb4e9 100644
--- a/arch/arm64/boot/dts/rockchip/rk3328-rock64.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3328-rock64.dts
@@ -49,6 +49,10 @@
 	model = "Pine64 Rock64";
 	compatible = "pine64,rock64", "rockchip,rk3328";
 
+	aliases {
+		ethernet0 = &gmac2io;
+	};
+
 	chosen {
 		// disable USB autosuspend as this solves USB stability issues
 		bootargs = "earlycon=uart8250,mmio32,0xff130000 swiotlb=1 kpti=0 no_console_suspend=1 usbcore.autosuspend=-1";
-- 
Armbian

